src/Client.php

```diff
@@ -49,6 +49,8 @@ public function __construct($wsdl, array $options = null)
      */
     public function __doRequest($request, $location, $action, $version, $one_way = 0)
     {
+        $request = $this->removeNameSpacedXsiObjectAttributeValues($request);
```

Comment:
It might be nice to have this be configurable as part of the class constructor. Do you think that's worth adding?

Reply:
there is no XML passed to the class consructor

Reply:
oh, configurable. Yeah, problem is we're not calling the constructor